cmd/kubelet/app/server.go

```diff
@@ -702,8 +701,6 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies, stopCh <-chan
 
 	utilruntime.ReallyCrash = s.ReallyCrashForTesting
 
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Where is this seeded now?

Reply:
this one @lavalamp ? https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/kubelet.go#L35